runtime/queries/metricsview_aggregation.go

```diff
@@ -11,6 +11,20 @@ import (
 	runtimev1 "github.com/rilldata/rill/proto/gen/rill/runtime/v1"
 	"github.com/rilldata/rill/runtime"
 	"github.com/rilldata/rill/runtime/drivers"
+	duckdbolap "github.com/rilldata/rill/runtime/drivers/duckdb"
+
+	databasesql "database/sql"
+	"database/sql/driver"
+	"sync"
+
+	"github.com/marcboeker/go-duckdb"
+	"github.com/rilldata/rill/runtime/pkg/pbutil"
+	"google.golang.org/protobuf/types/known/structpb"
+)
+
+var (
+	dbOnce    sync.Once
+	connector driver.Connector
```

Comment:
It's not safe to reuse a connector for multiple DBs. So it should be created at the same time as the `OpenDB` call, every time.